en/animation/edit-animation-clip.md

```diff
@@ -40,105 +40,21 @@ Next, let's take a look at the process of creating a Sprite animation as an exam
-
-### Copying/pasting keyframes
-
-Copy and paste keyframe data, support cross-node and cross-clip usage. The following two usage methods are included:
-
-- After selecting a keyframe (multiple selection is possible), use the shortcut keys <kbd>Ctrl</kbd> + <kbd>C</kbd> and <kbd>Ctrl</kbd> + <kbd>V</kbd> to copy and paste. Note that the shortcut paste will start at the position of the current time control line.
-- After selecting a keyframe (multiple selections are allowed), right-click on (any) keyframe and select **Copy Keyframe** in the popup menu, then right-click on the target animation property track and select **Paste Keyframe** to paste the keyframe on the current animation property track.
-
-The above two pasting methods differ when copying and pasting data from multiple property tracks, the difference mainly lies in:
-
-- When using shortcut keys to paste keyframes, the keyframes will be pasted on the animation property track where they were copied, in the order of the copied keyframes.
-
-  ![copy keyFrames1](edit-animation-clip/copy-keyframes2.gif)
-
-- When right-clicking on the target property track and select Paste, only the copied data will be pasted on the target property track.
-
-  ![copy keyFrames1](edit-animation-clip/copy-keyframes1.gif)
-
-Component property tracks (e.g.: `x`, `y`, `z` properties under the animation property track `position`) as separate property tracks will also follow this rule. If right-clicking on the target component track after copying the keyframe data on the main track (e.g.: animation property track `position`) to paste it, only the target component track will be pasted with the keyframe data. Be sure to copy the data according to the actual position you want to paste, to avoid unintended results.
-
-## Common operations with node data
+### Common operations with node data
 
 The animation clip defines the position of the animation data by the name of the node, itself ignoring the root node, and the rest of the child nodes find the corresponding data by the relative path index to the root node.
 
 Right-clicking on a node in the **Node List** area of the **Animation** panel is currently supported to choose between clearing node data, migrating node data, and copying and pasting node data.
 
 ![node operation](edit-animation-clip/node-operation.png)
 
-### Clearing node data
+#### Clearing node data
 
 Right-click the node that needs to clear all animation data (keyframes), then select **Clear data**, and click **Clear** in the popup window.
 
-### Migrating node data
+#### Migrating node data
 
 Right-click the node that needs to migrate all animation data, then select **Migrate Data**, the node will show a dynamic dashed box, when the mouse moves to other nodes, it will show the prompt of "Migrate data to this node", click it and select **Migrate** in the pop-up window. If there is no need to migrate, just select **Cancel**.
```

Comment:
```suggestion
Right-click the node that needs to migrate all animation data, then select **Move Data**, the node will show a dynamic dashed box, when the mouse moves to other nodes, it will show the prompt of "Move data to this node", click it and select **Move** in the pop-up window. If there is no need to migrate, just select **Cancel**.
```